behave/features/specialist_provider.feature

```diff
@@ -101,17 +101,31 @@ Scenario: Specialist Provider rejects a case
   Given I am on the specialist provider cases dashboard page
   And I select a case to reject from the dashboard
   And I am taken to the "specialist provider" case details page
-  And I select 'Reject'
+  And I select 'Reject' in the case details page
   Then the reject modal appears on screen
   And I select a reject reason of 'MIS-OOS'
   And I enter a reason into the Notes textarea
   When I select the 'Reject case' button
   Then I am on the specialist provider cases dashboard page
   And I confirm that my case has an Outcome code of 'MIS-OOS'
 
-# p13 split a case (LGA-1857)
+# p13 split a case (LGA-1857, LGA-1858)
 @specialist-provider-split-case
 Scenario: Specialist Provider rejects a case
   Given I am on the specialist provider cases dashboard page
   And I select a case to split from the dashboard
   And I am taken to the "specialist provider" case details page
+  And I select 'Split' in the case details page
+  Then the split case modal appears on screen
+  And the 'New case' drop down values are
+  | label           | value                                                                    |
+  | Category of law | Education                                                                |
+  | Matter type 1   | ESEN - Special educational needs and/or children unable to attend school |
+  | Matter type 2   | EDSC - School                                                            |
+  And I enter a comment into the new case notes textarea
+  And I select 'To operator for assignment' for the 'Assign' radio options
+  When I select the 'Split case' button
+  Then the message 'Case split successfully' appears on the case details page
+  And I select the 'Sign out' link
+  And that I am logged in as "CHS_GENERAL_USER"
```

Comment:
I am half tempted to suggest change the name of this to `I am logged in as ""` so it flows better with the and. Don't know if it is worth it though as is a global change?

Reply:
I think that's a good suggestion, will you be making the change?

Reply:
This will require quite a few changes, can this be done as it's own separate PR?

Reply:
ok, let's put this into the refactoring of end to end tests